index.html

```diff
@@ -459,146 +459,53 @@ <h4 id="additional-definitions-">Additional Definitions:</h4>
-            remove others</td>
-        </tr>
-        <tr>
-          <td>class relationships</td>
-          <td>unambiguous structure</td>
-          <td>limited cardinality, e.g. only one form per
-            operation per interaction affordance.</td>
-        </tr>
-        <tr>
-          <td>values of vocabulary terms</td>
-          <td>simplified processing</td>
-          <td>Limit the length of characters per string.
-            Always use arrays, where the spec permits a
-            string or an array of strings.</td>
-        </tr>
-        <tr>
-          <td>data schemas</td>
-          <td>simplified processing</td>
-          <td>No arbitrary nested objects or arrays of
-            arrays</td>
-        </tr>
-        <tr>
-          <td>security</td>
-          <td>reduced implementation effort</td>
-          <td>Only a restricted set of security
-            mechanisms</td>
-        </tr>
-        <tr>
-          <td>protocol binding</td>
-          <td>guaranteed protocol semantics</td>
-          <td>limited protocol(s) and protocol features,
-            Example: predefined mapping of http verbs (GET/PUT) to
-            operation verbs, similar constraints for other protocols.</td>
-        </tr>
-      </table>
-
-
-      <p>These constraints and rules fall into two categories:</p>
-      <ul>
-        <li>constraints for the data model.</li>
-        <li>constraints for the protocol binding.</li>
-      </ul>
-
-      <p>These two categories are orthogonal to each other,
-        i.e. a data model that conforms to a profile can be
-        mapped to different protocols. The protocol binding for
-        each protocol may contain additional (protocol-specific)
-        constraints.</p>
-
-      <p>A profile is not exclusive, i.e. a thing may conform
-        to multiple profiles. Profiles can build on top of each
-        other or overlap, extended profiles can be built on top
-        of the core profile.</p>
-      <p>
-        This specification does not put any requirements on the
-        scope and contents of other profiles.
-      </p>
-
-      <figure id="WoT-Profiles">
-        <img src="images/WoT Profiles.png" class="wot-profiles" alt="WoT Profiles" />
-        <figcaption>WoT Core Profile - Other Profiles</figcaption>
-      </figure>
-
-      <p>
-        In the present document, we define a <a>Core Profile</a>
-        by defining a <a>Core Data Model</a> and a set of <a href="#protocol-binding">Protocol Binding</a> Rules
-        for selected protocols.
-        <!--
-				In the present version of this document we define a mapping to
-				HTTP(S). It is envisioned, that future versions of this document
-				contain additional mapping to Web Sockets and other protocols.
-                -->
-      </p>
-    </section>
+    <pre class="example">
+    {
+      "@context": "https://www.w3.org/2019/wot/td/v1",
+      "id": "urn:dev:ops:32473-WoTLamp-1234",
+      "profile": [
```

Comment:
interesting. +1 to define [profile](url) as string and array of strings

Reply:
Just to be precise, the text above is read as if it should always be an array

Reply:
@sebastiankb The "array" AND "string" type definition is causing confusion now and then - as in the rest of the profile spec, I suggest to describe single value profile values always as arrays.

Reply:
@egekorkan Yes sorry that's unclear, I'll work on improving the wording.

@mlagally Given the vast majority of Things are likely to only conform to a single profile, I think that would be wasteful. It's only really complicated if it's not defined from the beginning, which then causes backwards compatibility problems in the future.

Reply:
@egekorkan OK, I've changed the wording, let me know if that's clearer?